nengo_spa/algebras/tvtb_algebra.py

```diff
@@ -204,3 +375,26 @@ def zero_element(self, d, sidedness=ElementSidedness.TWO_SIDED):
             Zero element.
         """
         return np.zeros(d)
+
+
+class TvtbSign(GenericSign):
+    def to_vector(self, d):
+        if self.sign is None:
+            raise NotImplementedError(
+                "There is no vector corresponding to an indefinite sign."
+            )
+        elif abs(self.sign) > 0:
+            return self.sign * TvtbAlgebra().identity_element(d)
```

Comment:
Since the two options are `+1` and `-1` it would be clearer IMO to call `identity_element` if it is `+1` and `negative_identity_element` if it is `-1`. This would be a bit more code, but it would deduplicate `negative_identity_element` in a sense.